packages/react-app/pages/index.jsx

```diff
@@ -87,56 +96,75 @@ export default function Index({ bgStats }) {
         description="A curated group of Ethereum builders creating products, prototypes, and tutorials to enrich the web3 ecosytem."
         image="/assets/bg_teaser.png"
       />
-      <Box>
-        <Box w={{ base: "100%", lg: "50%" }} pl={{ base: "50px", lg: "15%" }} pr="50px" py="50px">
-          <Box maxW="500px" margin={{ base: "auto", lg: "0" }} textAlign={{ base: "center", lg: "left" }}>
-            <Heading as="h1" mb="10px">
-              BuidlGuidl{" "}
-              <chakra.span fontWeight="500" fontSize={16}>
-                APP v3.5
-              </chakra.span>
-            </Heading>
-            <Text mb="25px" maxW="470px">
-              The BuidlGuidl ‘backoffice’, where you’ll be able to dig more into the day to day activity of the guild.
-            </Text>
-            {/*Builds / Builders / ETH distributed Ξ*/}
-            <Box d="inline-block">
-              <HStack mt="50px" justifyContent={{ base: "center", lg: "initial" }}>
-                <LinkBox>
-                  <StatBox
-                    value={bgStats.builderCount}
-                    monthlyValue={bgStats.buildersIncrementMonth}
-                    title="builders"
-                    link="/builders"
-                  />
-                </LinkBox>
-                <LinkBox>
-                  <StatBox
-                    value={bgStats.buildCount}
-                    monthlyValue={bgStats.buildsIncrementMonth}
-                    title="builds"
-                    link="/builds"
-                  />
-                </LinkBox>
-                <LinkBox onClick={() => smoothScroll(streamSection)} cursor="pointer">
-                  <StatBox
-                    value={`Ξ ${bgStats.streamedEth.toFixed(2)}`}
-                    monthlyValue={`Ξ ${bgStats.streamedEthIncrementMonth.toFixed(2)}`}
-                    title="streamed"
-                  />
-                </LinkBox>
-              </HStack>
-              <HStack mt="5px" justifyContent={{ base: "center", lg: "center" }}>
-                <Text fontSize="xs" color="green.500">
-                  ▲ Monthly change
-                </Text>
-              </HStack>
-            </Box>
-          </Box>
-        </Box>
+      {/* Hero*/}
```

Comment:
Question: Do you think we should start moving this to `components` or `components/home`?